Jenkinsfile

```diff
@@ -54,14 +54,16 @@ pipeline {
             steps {
                 script {
                     if (env.BRANCH_NAME == 'master') {
-                        docker.build("javaduck:${env.BUILD_ID}", "aggregation-worker/")
+                        docker.build("javaduck:${env.BUILD_TAG}", "aggregation-worker/")
                         withEnv(['PATH+EXTRA=/var/lib/jenkins/bin']) {
                             docker.withRegistry(env.ECR_REGISTRY_URL) {
-                                docker.image("javaduck:${env.BUILD_ID}").push("latest")
+                                dockerImage = docker.image("javaduck:${env.BUILD_TAG}")
+                                dockerImage.push("${env.BUILD_TAG}")
+                                dockerImage.push("latest")
                             }
                         }
                     } else {
-                        docker.build("javaduck-${env.BRANCH_NAME}:${env.BUILD_ID}", "aggregation-worker/")
+                        docker.build("javaduck-${env.BRANCH_NAME}:${env.BUILD_NUMBER}", "aggregation-worker/")
```

Comment:
Ok this is interesting now. This is now so close to the BUILD_TAG that I wonder if we shouldn't just be using the actual BUILD_TAG value.